webapp/portlet/src/main/webapp/WEB-INF/jsp/bannerLogo.jsp

```diff
@@ -58,11 +68,12 @@ document.addEventListener('spaceDetailUpdated', event => {
       <div class="container pa-0 ps-3">
         <div class="d-flex mx-0 pa-0">
           <% if (logoPath != null) { %>
-          <a id="UserHomePortalLink" href="<%=portalPath%>" class="pe-3 logoContainer">
+
+          <a id="UserHomePortalLink" href="<%=portalPath%>" class="pr-3 logoContainer <%=spaceAvatarClass%> " >
             <img src="<%=logoPath%>" class="<%=imageClass%>" alt="<%= logoTitle%> - Homepage">
           </a>
           <% } %>
-          <a href="<%=portalPath%>" title="<%=logoTitle%>" class="ps-2 align-self-center brandingContainer <%=titleClass%>">
+          <a href="<%=portalPath%>" title="<%=logoTitle%>" class="pl-2 align-self-center brandingContainer <%=titleClass%> <%=spaceAvatarClass%> ">
```

Comment:
Please pay attention to use `ps-*` classes instead of `pr-*` and `pl-*` to ensure to not break RTL support.

Reply:
okay I'll do !